prover/src/bin/prover_rpcd.rs

```diff
@@ -7,7 +7,7 @@ use prover::shared_state::SharedState;
 #[derive(Parser, Debug)]
 #[clap(version, about)]
 /// This command starts a http/json-rpc server and serves proof oriented methods.
-pub(crate) struct ProverdConfig {
```

Comment:
This only starts to make sense if moved to `src/config.rs` and being a public mod.

Reply:
I don't know why I thought you did this. I think I should change it back and then in anothe PR move it to its own module

Reply:
Ok for me